### ISSUE TITLE
Upgrade to v6.03 

### ISSUE THREAD

**Issue by xerudro** [ddavness/power-mailinabox#90]

**Comment by xerudro** [comment]
Today i tried to update from v6.02 to 6.03 using curl -L https://power-mailinabox.net/setup.sh | sudo bash, got the upgrade stuck on : Installing Mail-in-a-Box system management daemon... for more than 10 minutes.

**Comment by xerudro** [comment]
Tried again, after i do a reboot, with the same outcome. I use Hetzner server, and i have firewall and fail2ban active. @ddavness any solution ? What you suggest ?

**Comment by ddavness** [comment]
You mean 60.2 and 60.3 respectively - 6.02 and 6.03 do not exist 😅

I am unable to reproduce the problem, however (that is, mine doesn't get stuck for 10 minutes). For all we know, it could be one of two things:
- Intermittent datacenter problems, which I find unlikely to happen (or Hetzner would have put up a notice about it already);
- The SSH connection decided to go to sleep, but the setup is still continuing on the background - in that case try pressing Enter and see if the rest of the setup magically happens in half a second.

**Comment by xerudro** [comment]
I managed to upgrade it, i stopped fail2ban and tried again but still had to wait few minutes 6-7 minutes until the upgrade ended. Not sure what was the culprit but i managed to make it work. But i`ve noticed something that before 60.2 i didnt , now on the status check i get messages about IPv6. Should i ignore them ? Or should i try to fix them ? I am using your fork with Cloudflare.

**Comment by ddavness** [comment]
Awesome. To be fair, there's a compile step on the Management Daemon but it shouldn't have taken that long to finish (30 seconds at worst).

Are you by any chance running this on Ubuntu 22.04? I've seen some folks complain about `fail2ban` on the upstream project.

**Comment by xerudro** [comment]
Yes, i run  it Ubuntu 22.04. Then i need to find a fix for the IPv6 DNS problems, and the one with SSH on IPv6, even if i dont understand why does it needs access SSH on IPv6 .


Here are the messages without the IPv6 

HTTP Web (nginx) is running (and available over IPv4 and the local IPv6 address), but it is not publicly accessible at
SSH Login (ssh) is running (and available over IPv4 and the local IPv6 address), but it is not publicly accessible at...:22.
Public DNS (nsd4) is running and available over IPv4 but is not accessible over ... port 53
Incoming Mail (SMTP/postfix) is running (and available over IPv4 and the local IPv6 address), but it is not publicly accessible at ...:25
Outgoing Mail (SMTP 465/postfix) is running (and available over IPv4 and the local IPv6 address), but it is not publicly accessible at ...:465
Outgoing Mail (SMTP 587/postfix) is running (and available over IPv4 and the local IPv6 address), but it is not publicly accessible at...:587...

and the list continues for Sieve, Dovecot, and HTTPS nginx.

**Comment by xerudro** [comment]
@ddavness if i will disable the IPv6 function will affect the functionality of the mail server ?

**Comment by ddavness** [comment]
Public IPv6? Probably not. However, the system keeps local IPv6 enabled (::1) to ensure `nsd` doesn't break.

**Comment by xerudro** [comment]
I looked up how to add ipv6 to ufw allowance rules and it seems a bit
complicated. And i mistaken it's not Ubuntu the distro. It's Debian
Bullseye.

mie., 2 nov. 2022, 11:51 David Duque ***@***.***> a scris:
